src/wiki/ventoy.md

```diff
@@ -0,0 +1,47 @@
+# Creating a Multi-OS Bootable Drive (with Ventoy)
+In this article, you will learn how to create a multi-OS bootable USB drive using Ventoy.
+
+Ventoy is a tool that allows you to create a bootable USB drive with multiple operating systems or recovery tools. Unlike other software, Ventoy allows you to simply store the ISO files **directly** on the USB. This can be really helpful if you have a suite of tools that you often use and you don't want to use a different USB drive for each one of them.
+
+## Installing Ventoy on a USB
+In this section, you will learn how to use the Ventoy-to-USB tool to install Ventoy on your USB.
+
+::: warning Caution
+Installing Ventoy **will wipe all data on the target drive**. Back up your data before proceeding, or use a drive that you know is empty.
+:::
+
+1. Download the [Ventoy-to-USB tool](https://www.ventoy.net/en/download.html). Since Ventoy is open-source, [you can also download the latest release from GitHub](https://github.com/ventoy/Ventoy/releases).
+
+    ![Download page of Ventoy](<img/ventoy/Ventoy download.png>)
+
+2. Plug-in the drive you want to use, which should have at least 8GB available.
+
+3. Right click on the file that you have just downloaded and select extract. Then, inside the folder named `ventoy-1.x.xx`, run the executable named `Ventoy2Disk.exe`. A UAC prompt will pop-up, click yes.
```

Comment:
Ambiguous instruction, you should state to right-click on the file in explorer

Something like 
```suggestion
3. In File Explorer, right-click on the file that you have just downloaded and select Extract. Then, inside the folder named `ventoy-1.x.xx`, run the executable named `Ventoy2Disk.exe`. A UAC prompt will pop up, click Yes.
```